src/core/Akka.Remote.Tests/RemotingSpec.cs

```diff
@@ -161,12 +158,77 @@ public void Remoting_must_support_remote_lookups()
             //TODO: using smaller numbers for the cancellation here causes a bug.
             //the remoting layer uses some "initialdelay task.delay" for 4 seconds.
             //so the token is cancelled before the delay completed.. 
-            var msg = await here.Ask<Tuple<string,IActorRef>>("ping", TimeSpan.FromSeconds(1.5));
+            var msg = await here.Ask<Tuple<string, IActorRef>>("ping", TimeSpan.FromSeconds(1.5));
             Assert.Equal("pong", msg.Item1);
             Assert.IsType<FutureActorRef>(msg.Item2);
         }
 
         [Fact]
+        public void Remoting_must_not_send_remote_recreated_actor_with_same_name()
```

Comment:
Guarantees that both remote `ActorSystem` instances respect the UIDs for actors discovered via remote messaging, and that old `IActorRef`s pointing to the dead actor now get their messages delivered to `DeadLetters`.